File: homelibrary/src/main/java/com/ren/MainActivity.java
package com.ren;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.alibaba.android.arouter.launcher.ARouter;
import com.ren.middleware.Module;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ARouter.getInstance().build(Module.HOME).navigation();
        finish();
    }
}

File: kotlin_module/src/main/java/com/ren/kotlin_module/MainActivity.kt
package com.ren.kotlin_module

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.launcher.ARouter
import com.ren.middleware.Module

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ARouter.getInstance().build(Module.KOTLIN).navigation()
        finish();
    }
}